admin-ui/app/routes/Dashboards/DashboardPage.tsx

```diff
@@ -310,15 +310,24 @@ function DashboardPage() {
       )
     } else navigate('/logout')
   }
+  const showModal = () => {
+    if (!hasFetchUserInformation) {
+      if (!access_token || !hasBoth(permissions, STAT_READ, STAT_JANS_READ)) {
+        return (
+          <GluuPermissionModal
+            handler={() => {
+              handleLogout()
+            }}
+            isOpen={true}
+          />
+        )
+      }
+    }
+  }
 
   return (
     <GluuLoader blocking={loading}>
-      <GluuPermissionModal
-        handler={() => {
-          handleLogout()
-        }}
-        isOpen={!access_token || !hasBoth(permissions, STAT_READ, STAT_JANS_READ)}
-      />
+      {showModal()}
```

Comment:
Consider aligning permission checks with Cedar across the Dashboard.

MAU moved to Cedar permissions; Dashboard still uses hasBoth(authReducer.permissions,...). To prevent drift, migrate to useCedarling and cedarPermissions here as well.